core/frontend/src/IModelApp.ts

```diff
@@ -269,6 +271,10 @@ export class IModelApp {
   public static get hasRenderSystem() { return this._renderSystem !== undefined && this._renderSystem.isValid; }
   /** The [[UiAdmin]] for this session. */
   public static get uiAdmin() { return this._uiAdmin; }
+  /** If the app is a NoRenderApp
+  * @internal
+  */
+    public static get noRender() { return this._noRender; }
```

Comment:
What is the motivation for this accessor? Nobody currently calls it and ideally no one will need to.